### PR TITLE
Add ADR for how to handle programme options redirects

### PR DESCRIPTION
**Pull request by BPScott**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: docs/arch/adr-003-serving-programme-options-redirects.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Currently Symfony supports supressing the logging of 404 errors when using the
+`fingers_crossed` error handler. This is done through "Activation Strategy"
+classes that control if an exception should trigger a log message. We will have
+to create a custom activation strategy that supresses the logging of 3xx
+exceptions in addition to 404 exceptions. If https://github.com/symfony/symfony/pull/25533
+is merged upstream then this custom activation strategy can be removed and
+replaced with configuration.
+
+Status
```
Comment (votemike) [inline docs/arch/adr-003-serving-programme-options-redirects.md]: What's this bit?

File: docs/arch/adr-003-serving-programme-options-redirects.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+------
+
+Accepted
+
+
+Consequences
+------------
+
+* Throwing a `ProgrammeOptionsRedirectHttpException` shall result in 3xx
```
Comment (votemike) [inline docs/arch/adr-003-serving-programme-options-redirects.md]: "shall result in 3xx response." => "shall result in a 3xx response."?

File: docs/arch/adr-003-serving-programme-options-redirects.md (hunk 1 above)
Comment (BPScott) [reply docs/arch/adr-003-serving-programme-options-redirects.md]: As per the ADR folder's README.md, read http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions for info on what the various sections are for.

In this case status is "Has this idea been accepted or is it still a WIP proposal". As I wouldn't merge this PR untill it has been accepted then there's not much point to have it be anything but accepted.

**Pull request merged**